theme/hessen/static/images/logo/HPI-Schul-Cloud-Bildmarke_transparent.svg

```diff
@@ -0,0 +1 @@
0 0 1-1.43-.335c-.998.877-2.315 1.58-4.381 1.58-1.086 0-1.69-.397-2.81-.397-.485 0-.935.086-1.356.473-.321-.048-.575-1.131-.14-1.626a1.964 1.964 0 0 1-1.747-1.247c.585-.25.662-.602.948-1.264.447-1.04 1.385-1.928 2.296-2.057-.09-.257-.128-.627-.128-1.076 0-.647.4-1.22.766-1.516l.077.07c.405.366.774.628 1.754.633h.031v-1.049c.335 0 1.126.839 2.276.839h.032l.031-.001c.476-.017.936-.22.936-.678 0-.481-.295-.572-.468-.572-.294 0-.321.303-.688.303-.374 0-.718-.268-.718-.565 0-.22.15-.335.477-.335.326 0 .545.09.84.09h.059c.333-.012 1.078-.126 1.078-.849 0-1.23-1.2-1.656-1.623-1.622.045.332-.23 1.052-1.008 1.052-.546 0-1.073-.436-1.073-.943 0-.172.023-.37.115-.491l.022-.015.029-.02a4.65 4.65 0 0 0 .492-.404l.013-.013-.113-.126-.047.043-.05.042a2.35 2.35 0 0 1-.441.321.556.556 0 0 1-.044-.25c0-.454.78-1.088 1.486-1.504.199-.494 1.144-1.538 2.98-1.538.282-.22.632-.27.992-.273h.023c.082 0 .185.006.313.031l-.048.04-.138.115c-.108.091-.236.2-.347.307l-.018.017.137.163.058-.05.303-.267c.127-.111.248-.215.32-.267l.037-.024c.411-.274.593-.289.796-.172zM12.907 8.587c.293 0 .757.167.757.543 0 .375-.2.798-.204 1.096-.011.72.782 1.213 1.717 2.535.324.46 2.03 1.49 3.113 2.053l.04.02a1.92 1.92 0 0 0-.106.618v.034c.002.339.024.622.071.853l.007.034-.008.002c-.886.235-1.766 1.076-2.217 2.099l-.029.066-.161.38-.028.064a2.237 2.237 0 0 1-.01.022l-.017.039a1.815 1.815 0 0 1-.135.244l-.002-.052-.002-.043-.003-.033c-.02-.235-.085-.453-.39-.961-.14.422-.702.608-1.088.545.286-.79.339-1.62-.02-2.665-.073.561-.405.658-.773.658v-.047l-.002-.046v-.046c-.023-.754-.205-1.708-1.12-2.948a.538.538 0 0 1-.429.207c-.807 0-1.264-.8-1.22-1.318l-.034-.005c-.57-.084-1.07-.356-1.07-1.047 0-.556.437-.713.76-.716h.01c.324 0 .527.05.796.203-.102-.375-.278-.502-.438-.607l-.097-.062c-.214-.137-.415-.282-.415-.593 0-.41.564-.834.918-.852.349-.015.649.232.657.576v.014c0 .343.301.731.61 1.067l.007-.049c.033-.231.053-.512-.002-.714-.044-.162-.169-.44-.154-.752.017-.313.372-.416.71-.416zm-2.543 4.948c.203.178.46.366 1.004.36a1.078 1.078 0 0 1-.874.477c-.408 0-.72-.365-.81-.526.418-.083.511-.172.68-.311zm-.863-2.512c-.131.424-.002.789.183 1.029-.192.086-.404.299-.49.646-.149-.172-.352-.457-.352-.82 0-.452.31-.788.66-.855zm10.254-.837c.119 0 .243.032.333.109-.192.247-.292.486-.292.785 0 .454.426 1.018 1.517 1.018.558 0 .78-.092 1.176-.212.395-.12.915-.308 1.484-.308.46 0 .659.02.828.046a1.14 1.14 0 0 1 .031.439c-.243-.06-.441-.103-.988-.103-.305 0-.793.118-1.142.247-.349.13-1.13.49-1.95.49-.896 0-1.805-.476-1.805-1.506 0-.671.432-1.005.808-1.005zM9.732 8.512c.345 0 .613.098.794.342-.515-.026-.948.278-1.144.495a1.21 1.21 0 0 0-.54-.126c.052-.351.372-.711.89-.711zm13.422-.63a.6.6 0 0 0-.139.356c0 .221.181.467.468.467.2 0 .397-.187.447-.316.114.102.224.277.224.277l.114-.086a.448.448 0 0 1 .036-.025c-.351-.465-.808-.644-1.15-.673zM12.19 7.674c.448 0 .71.238.863.633a1.73 1.73 0 0 0-.486-.048.926.926 0 0 0-.659.253c-.114-.184-.26-.302-.54-.453.137-.214.386-.385.822-.385z" fill="#FFF" fill-rule="evenodd"/></svg>
```

Comment:
Ich würde einen anderen Filename vorschlagen, sonst bleibt das so falls der Prototyp dann übernommen wird ;)

Reply:
Auch wenn wir es hier nur überschreiben wollen, sollten wir überlegen das einfach nur Bildmarke_transparent.svg im ganzen client zu nennen

Reply:
Also der Dateiname ist von Main theme vorgegeben, daher war da keine Auswahl. Aber ja, die Bildnamen in diesem Client sind furchtbar und sollten Mal überarbeitet werden.